pdr_backend/lake/duckdb_data_store.py

```diff
@@ -349,3 +351,101 @@ def get_query_move_table_data(self, from_table, to_table):
         return (
             f"INSERT INTO {to_table.table_name} SELECT * FROM {from_table.table_name};"
         )
+
+    @enforce_types
+    def export_tables_to_parquet_files(
+        self,
+        seconds_between_exports: int,
+        number_of_files_after_which_combine_into_one: int,
+    ):
+        export_folder_path = f"{self.base_path}/exports"
+        tables = self.query_data("SHOW TABLES")["name"]
+
+        # Ensure export folder exists
+        os.makedirs(export_folder_path, exist_ok=True)
+
+        for table in tables:
+            table_folder_path = os.path.join(export_folder_path, table)
+            os.makedirs(table_folder_path, exist_ok=True)
+
+            if not self._should_export(table_folder_path, seconds_between_exports):
+                continue
+
+            self._export_table_to_parquet(table, table_folder_path)
+
+            if self._should_combine_files(
+                table_folder_path, number_of_files_after_which_combine_into_one
+            ):
+                self._combine_parquet_files(table_folder_path)
+
+    def _should_export(
+        self, table_folder_path: str, seconds_between_exports: int
+    ) -> bool:
+        max_timestamp_from_parquet = self._get_max_timestamp_from_parquet_files(
+            table_folder_path
+        )
+        current_timestamp = UnixTimeS(int(time.time())).to_milliseconds()
+
+        return (
+            current_timestamp - max_timestamp_from_parquet
+            >= seconds_between_exports * 1000
+        )
+
+    def _get_max_timestamp_from_parquet_files(self, table_folder_path: str) -> int:
+        try:
```

Comment:
So, what this means is that for bronze tables we have to re-export the entire table each time because as a saved row changes we can't just update the table. Am I right?